index.html

```diff
@@ -540,8 +540,11 @@ <h3>Enterprise</h3>
             <h2>Group 1</h2>
             <div class="sub-footer-link" role="link">
               <a href="index.html">Home</a>
+              <br>
               <a href="index.html#feature">Features</a>
+              <br>
               <a href="index.html#pricing">Pricing</a>
+              <br>
```

Comment:
This is _okay_, but it might be better to add styling for the links instead i.e. display block or flex.